draft-rosa-bmwg-vnfbench

```diff
@@ -1110,19 +1141,7 @@ Internet-Draft                  VNFBench                   December 2018
    also VNF Benchmarking Descriptor (VNF-BD) examples on the vIMS and
    OVS targets as described in [Rosa-b] and [Rosa-c].
```

Comment:
How about the code availability. I think we really need this when we hand in the updated draft.

Reply:
Solved!